app-admin/filebeat/files/filebeat.confd

```diff
@@ -0,0 +1,5 @@
+#FILEBEAT_USER=""
```

Comment:
If this is meant to be configured by the user, adding some documentation and explaining what these variables are about would be useful.